plasmapy/analytic/parameters.py

```diff
@@ -0,0 +1,852 @@
+        specific heats, Poisson constant, or isentropic expansion parameter).
+        Defaults to 5/3 which is appropriate for a monatomic gas.
+
+    Returns
+    -------
+    V_S : Quantity
+        The ion sound speed.
+
+    Example
+    -------
+    >>> from astropy import units as u
+    >>> ion_sound_speed(5e6*u.K)
+    <Quantity 203155.10435273056 m / s>
+    >>> ion_sound_speed(1*u.keV)
+    <Quantity 309496.9076337826 m / s>
+
+    """
+
+    if gamma < 1:
+        raise ValueError("The ratio of specific heats cannot be less than one")
+    elif gamma is np.inf:
+        return np.inf*u.m/u.s
+    
+
+    try:
+        m_i = ion_mass(ion)
+    except Exception:
+        raise ValueError("Unable to find ion mass")
+    
+    try:
+        T_i = T_i.to(u.K, equivalencies=u.temperature_energy())
+    except Exception:
+        raise u.UnitConversionError("The ion temperature in ion_sound_speed "
+                                    "cannot be converted to Kelvin.")
+
+    try:
+        V_S = (np.sqrt(gamma*k_B*T_i/m_i)).to(u.m/u.s)
+    except Exception:
+        raise ValueError("Unable to find ion sound speed")
+
+    if V_S > c:
+        raise UserWarning("Ion sound speed is greater than speed "
+                          "of light")
+
+    return V_S
+
+
+def electron_thermal_speed(T_e):
+    """Returns the most probable speed for an electron within a
+    Maxwellian distribution.
+
+    Parameters
+    ----------
+    T_e : Quantity
+        The electron temperature in either Kelvin or electron-volts
+
+    Returns
+    -------
+    V_Te : Quantity
+        Electron thermal speed
+
+    Notes
+    -----
+    The electron thermal speed is given by:
+
+    .. math::
+    V_{th,e} = \sqrt{\frac{2 k_B T_e}{m_e}}
+
+    This function yields the most probable speed within a distribution
+    function.  However, the definition of thermal velocity varies by
+    the square root of two depending on whether or not this velocity
+    absorbs that factor in the expression for a Maxwellian
+    distribution.  In particular, the expression given in the NRL
+    Plasma Formulary [1] is a square root of two smaller than the
+    result from this function.
+
+    Examples
+    --------
+    >>> electron_thermal_speed(5*u.eV)
+    <Quantity 1326205.1454609886 m / s>
+    >>> electron_thermal_speed(1e6*u.K)
+    <Quantity 5505694.743141063 m / s>
+
+    """
+
+    try:
+        T_e = T_e.to(u.K, equivalencies=u.temperature_energy())
+        V_Te = (np.sqrt(2*k_B*T_e/m_e)).to(u.m/u.s)
+    except Exception:
+        raise ValueError("Cannot find electron thermal speed")
+
+    if V_Te > c:
+        raise UserWarning("Electron thermal speed is greater than speed "
+                          "of light")
+
+    return V_Te
+
+
+def ion_thermal_speed(T_i, ion='p'):
+    """Returns the most probable speed for an ion within a Maxwellian
+    distribution.
+
+    Parameters
+    ----------
+    T_i : Quantity
+        The ion temperature
```

Comment:
`The ion temperature **in either Kelvin or electron-volts**`

For consistency with other docstrings.